[PL-51268] PL 1.38.x Release Notes (#6726)
harness/developer-hub: release-notes/platform.md

## May 2024

### Version 1.38.2<!--  May 20, 2024 -->

#### Fixed issues

> Invalid user search results were returned when querying from page `2` or higher. We have updated the search functionality to reset the `pageIndex` to `0` after a search query is added or updated, ensuring accurate search results even when the `pageIndex` is `2` or higher. (PL-50907, ZD-62990)

Reviewer: ```suggestion
Invalid user search results were returned when querying from page `2` or higher. Harness updated the search functionality to reset the `pageIndex` to `0` after a search query is added or updated, ensuring accurate search results even when the `pageIndex` is `2` or higher. (PL-50907, ZD-62990)
```

> Delegate registration was not failing for inactive accounts. We have added a check during delegate registration to verify account status. Delegates will now fail to register for accounts marked as DELETED or INACTIVE. This item requires Harness Delegate version 24.04.83000. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-48082)

Reviewer: ```suggestion
Delegate registration was not failing for inactive accounts. Harness added a check during delegate registration to verify account status. Delegates will now fail to register for accounts marked as `DELETED` or `INACTIVE`. This item requires Harness Delegate version 24.05.83001. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-48082)
```
